tests/unit_tests/test_tethys_cli/test_gen_commands.py

```diff
@@ -73,13 +73,13 @@ def test_generate_command_apache_option(
         mock_args.type = GEN_APACHE_OPTION
         mock_args.directory = None
         mock_os_path_isfile.return_value = False
-        mock_settings.side_effect = ["/foo/workspace", "/foo/static"]
+        mock_settings.side_effect = ["/foo/workspace", "/foo/static", "/foo/media"]
 
         generate_command(args=mock_args)
 
         mock_os_path_isfile.assert_called_once()
         mock_file.assert_called()
-        mock_settings.assert_called_with("STATIC_ROOT")
```

Comment:
Is there a reason you're dropping the check for STATIC_ROOT instead of doing both?

Reply:
whoops, added it back in.